src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,105 @@ describe("MarketingCollections", () => {
       curatedMarketingCollections: payload,
     })
   })
+
+  it("uses the curated sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockReturnValueOnce(Promise.resolve({ body: payload }))
+
+    const data = await runQuery(query, context)
+
+    expect(context.marketingCollectionsLoader).toHaveBeenCalledWith({
+      size: 2,
+      slugs: [
+        "new-from-tastemaking-galleries",
+        "new-from-nonprofits-acaf27cc-2d39-4ed3-93dd-d7099e183691",
+        "new-from-small-galleries",
+        "new-from-leading-galleries",
+        "new-to-artsy",
+      ],
+    })
```

Comment:
**suggestion**: It reads like this test is focused on the switch from category+sort to slugs. I think that would be better accomplished with something like the following:

```ts
 expect(marketingCollectionsLoaderMock.mock.calls[0][0]).not.toContainKeys([
    "category",
    "sort",
  ])
  expect(marketingCollectionsLoaderMock).toHaveBeenCalledWith(
    expect.objectContaining({
      slugs: [
        "art-under-500-dollars",
        "art-under-1000-dollars",
        "art-under-2500-dollars",
        "art-under-5000-dollars",
        "art-under-10000-dollars",
        "art-under-25000-dollars",
        "art-under-50000-dollars",
      ],
    })
  )
})
```

Reply:
Okay, sounds good, adding the test case for arguments.